docs/DeveloperGuide.md

```diff
@@ -186,14 +187,19 @@ Location: `seedu.duke.ui`
 __Note:__
 * `CommandResult` : Contains all the information we want to display to the user.
 * `Command` : Abstract class that tells us whether we are in the main level or in a workout.
+* 
 * `Ui` : Handles the display of information to the user.
 
 In the Ui class, `withIndent` is true if the user is in `workoutMode`, false otherwise.
 
+##Logging
+
+* [Logger Guide](Logging.md)
+
```

Comment:
Is it compulsory to link to another file instead of just writing it down here?

Reply:
In this part, we could add more guides such as documentation guide and testing guide and so on. If we link to separate docs can concurrently work on them. (Idk man I just copied address book level 3).

Reply:
Oh if we are adding other guides then sure